decidim/decidim
# Fix broken conference registering when no registration types

#### :tophat: What? Why?
Fix conferences registers when the registration types are empty and registration is enabled

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #12704 

#### Testing
1. go to admin dashboard
2. create a conference with registrations enabled and without registration_types
3. publish the conference
4. click on See conference button
5. click on Register button
6. can access to the registration page

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

co-writter: @Stef-Rousset 

:hearts: Thank you!
